pkg/controllers/disruption/drift_test.go

```diff
@@ -727,6 +728,48 @@ var _ = Describe("Drift", func() {
 			Expect(ExpectNodes(ctx, env.Client)).To(HaveLen(1))
 			ExpectNotFound(ctx, env.Client, nodeClaim, node)
 		})
+		It("should delete drifted nodes when they are empty and consolidation is disabled", func() {
```

Comment:
We should add a test that shows the ordering of disruption given non-empty nodes are being prioritized.